# input topic name setting ===========> Please change according to your dataset
lidar_topic: "/camera/rgb/points"

pose_source: 0 # [0:tf_tree, 1:tf_topic, 2:odom_topic]
tf_topic: ""

timestamp_tolerance_: 1
world_frame: "world"
child_frame: "openni_rgb_optical_frame"

debug_print: true
open3d_vis: false
vis_rate: 0.1
# when -1 means no filter on range
min_scan_range: -1 # Ignore points closer than this value (meters) (default 0.4)
max_scan_range: 10.0 # Ignore points far than this value (meters) (default 12.0)
max_height: 20.0 # ignore points higher than this value default 5.0

# Check on paper
sdf_space_carving: true

sdf_trunc: 0.07

# voxel resolution (meters)
sdf_voxel_size: 0.01

# min weight setting TODO add details description
sdf_min_weight: 3.0
sdf_deactivate: 0.05

# Translation
tx: 0
ty: 0
tz: 0
# Rotation Quaternion
x: 0
y: 0
z: 0
w: 1
